test/unit/mem/buf/src/BufTest.cpp

```diff
@@ -64,6 +178,40 @@ TEMPLATE_LIST_TEST_CASE("memBufZeroSizeTest", "[memBuf]", alpaka::test::TestAccs
     testBufferMutable<Acc>(extent);
 }
 
+TEMPLATE_LIST_TEST_CASE("memBufAsyncBasicTest", "[memBuf]", alpaka::test::TestAccs)
+{
+    using Acc = TestType;
+    using Dim = alpaka::Dim<Acc>;
+    using Idx = alpaka::Idx<Acc>;
+
+    if constexpr(isAsyncBufferSupported<Acc>())
+    {
+        auto const extent
+            = alpaka::createVecFromIndexedFn<Dim, alpaka::test::CreateVecWithIdx<Idx>::template ForExtentBuf>();
+        testAsyncBufferMutable<Acc>(extent);
+    }
+    else
+    {
+        printAsyncBufferUnsupported<Acc>();
```

Comment:
```suggestion
        INFO("Stream-ordered memory buffers not supported");
```
And delete `printAsyncBufferUnsupported`. I think this is good enough. But do as you like.

Two more occurences below.

Reply:
OK for me.